lib/Db/ContextNodeRelationMapper.php

```diff
@@ -28,6 +28,26 @@ public function deleteAllByContextId(int $contextId): void {
 		$qb->executeStatement();
 	}
 
+	public function getRelIdsForNode(int $nodeId, int $nodeType): array {
+		$qb = $this->db->getQueryBuilder();
+		$qb->select('id')->from($this->table)->where($qb->expr()->eq('node_id', $qb->createNamedParameter($nodeId)))
+		->andWhere($qb->expr()->eq('node_type', $qb->createNamedParameter($nodeType)));
```

Comment:
Just for easier reading:

```suggestion
		$qb->select('id')->from($this->table)
			->where($qb->expr()->eq('node_id', $qb->createNamedParameter($nodeId)))
			->andWhere($qb->expr()->eq('node_type', $qb->createNamedParameter($nodeType)));
```